blocks.js

```diff
@@ -7553,13 +7556,43 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-            // } else if (key.indexOf('§_def') === 0) {
-            //     menu.addItem(choices[key].blockInstance(), choices[key]);
+         // } else if (key.indexOf('§_def') === 0) { 
+         //     menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charAt(key.length - 1) == '{') {
+                // Submenu
+                // 0x25ba = Unicode "BLACK RIGHT-POINTING POINTER"
+                // (not to be confused with "black right-pointing TRIANGLE")
+                menu.addItem(key.substring(0,key.length - 1).
+							 concat(String.fromCharCode(0x25ba)),
```

Comment:
Also, why use Morphic's `concat` and not just `String::concat` or `+`.  Also I think you should be able to simply write `'\u25BA' ` instead of calling fromCharCode. (And the `\u` is a pretty clear unicode indicator.

Would it make sense to call localize around this character? I don't know if any other language would want it to be different?